### ISSUE TITLE
[BUG] Authentication is not working

### ISSUE THREAD

**Issue by SubhamB2003** [UniKonf/Vibey#488]
### Description of the bug

Hey, if I click the Google login button and then back then it redirects to the dashboard page but I am not login. I think this issue needs to fix.
Can you please assign it to me?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Deepu178** [comment]
@SubhamB2003 how are you gonna fix this issue? Can you please explain a little bit?

**Comment by SubhamB2003** [comment]
@Deepu178 Yes if I am successful in login then only it redirects to the dashboard page otherwise it stays on the sign in page.
Can you please assign it to me?

And another thing is this shows a password button on the sign-up is not added till now. If this issue is not assigned to anyone can I create a new issue related to this option?

**Comment by Deepu178** [comment]
@SubhamB2003 how are you gonna fix it?

**Comment by SubhamB2003** [comment]
> @SubhamB2003 how are you gonna fix it?

I changed the fallback URL to "/" currently it was "/dashboard"

**Comment by Deepu178** [comment]
@SubhamB2003 is it working fine locally?
